drivers/edac/shell.c

```diff
@@ -62,6 +62,7 @@ static int cmd_edac_info(const struct shell *shell, size_t argc, char **argv)
 {
 	const struct device *dev;
 	uint64_t error;
+	int ret;
```

Comment:
Please use `err` for simple `int` type errors. If you think this will be confusing with your `error` variable, then rename that one e.g. to `ecc_err` or something similar.

Reply:
done